src/components/Footers.tsx

```diff
@@ -60,7 +66,52 @@ function AboutDialog() {
             </Flex>
           </DialogBody>
           <DialogFooter>
-            <Button onClick={() => setVisible(false)}>Close</Button>
+            <DialogActionTrigger asChild>
+              <Button>Close</Button>
+            </DialogActionTrigger>
+          </DialogFooter>
+        </DialogContent>
+      </DialogRoot>
+    </>
+  );
+}
+
+function PrivacyPolicyDialog() {
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <>
+      <Link onClick={() => setVisible(true)} colorPalette="blue">
+        Privacy Policy
+      </Link>
+      <DialogRoot
+        lazyMount
+        open={visible}
+        onOpenChange={() => setVisible(false)}
+      >
+        <DialogContent>
+          <DialogHeader>
+            <DialogTitle>Privacy Policy</DialogTitle>
+          </DialogHeader>
+          <DialogBody>
+            <Flex direction="column" gap={4}>
+              <Text>
```

Comment:
I just copy-pasted this from what was previously in privacy.html, but most of it isn't really applicable since we don't currently have GCal integration. Still, good to leave in for whenever we fix it I suppose